front/app/components/admin/Graphs/LineChart/index.tsx

```diff
@@ -94,6 +94,7 @@ const LineChart = <Row,>({
           DEFAULT_LEGEND_OFFSET
         )}
         ref={innerRef}
+        accessibilityLayer
```

Comment:
c

Reply:
Did you also test them with a screen reader?

Reply:
I would warmly encourage you to test a few in our app before sending it back to the auditor.